### ISSUE TITLE
'TimeSpanHumanize_MultipleMilliseconds' was not found

### ISSUE THREAD

**Issue by aleksmelnyk** [Geodan/pg2b3dm#118]
hi there,

on non english pc it raises following error:

```
Unhandled exception. System.ArgumentException: The resource object with key 'TimeSpanHumanize_MultipleMilliseconds' was not found (Parameter 'resourceKey')
   at Humanizer.Localisation.Formatters.DefaultFormatter.Format(String resourceKey, Int32 number, Boolean toWords) in /_/src/Humanizer/Localisation/Formatters/DefaultFormatter.cs:line 137
   at Humanizer.Localisation.Formatters.DefaultFormatter.GetResourceForTimeSpan(TimeUnit unit, Int32 count, Boolean toWords) in /_/src/Humanizer/Localisation/Formatters/DefaultFormatter.cs:line 102
   at Humanizer.Localisation.Formatters.DefaultFormatter.TimeSpanHumanize(TimeUnit timeUnit, Int32 unit, Boolean toWords) in /_/src/Humanizer/Localisation/Formatters/DefaultFormatter.cs:line 71
   at Humanizer.TimeSpanHumanizeExtensions.CreateTheTimePartsWithUpperAndLowerLimits(TimeSpan timespan, CultureInfo culture, TimeUnit maxUnit, TimeUnit minUnit, Boolean toWords) in /_/src/Humanizer/TimeSpanHumanizeExtensions.cs:line 63
   at Humanizer.TimeSpanHumanizeExtensions.Humanize(TimeSpan timeSpan, Int32 precision, Boolean countEmptyUnits, CultureInfo culture, TimeUnit maxUnit, TimeUnit minUnit, String collectionSeparator, Boolean toWords) in /_/src/Humanizer/TimeSpanHumanizeExtensions.cs:line 50
   at Humanizer.TimeSpanHumanizeExtensions.Humanize(TimeSpan timeSpan, Int32 precision, CultureInfo culture, TimeUnit maxUnit, TimeUnit minUnit, String collectionSeparator, Boolean toWords) in /_/src/Humanizer/TimeSpanHumanizeExtensions.cs:line 33
   at pg2b3dm.Program.<>c__DisplayClass4_0.<Main>b__0(Options o) in D:\dev\github.com\geodan\pg2b3dm\src\pg2b3dm\Program.cs:line 251
   at CommandLine.ParserResultExtensions.WithParsed[T](ParserResult`1 result, Action`1 action)
   at pg2b3dm.Program.Main(String[] args) in D:\dev\github.com\geodan\pg2b3dm\src\pg2b3dm\Program.cs:line 29
```
Looks like Humanizer issue?

**Comment by bertt** [comment]
I've created version 1.8.2, can you try?

**Comment by aleksmelnyk** [comment]
fixed in 1.8.2,

thanks much.
